src/components/shared/nav/nav.module.scss

```diff
@@ -0,0 +1,107 @@
+@use '@variables' as *;
+
+.container {
+  position: sticky;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  top: 0;
+  width: 100%;
+  height: 120px;
+  padding: 0 1rem 0 1rem;
+  background-color: $white;
+  font-family: open-sans;
+  z-index: 100;
+}
+
+.maxWidth {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  max-width: 1131px;
+}
+
+.navigation {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  justify-content: space-between;
+}
+
+.navUl {
+  display: flex;
+  gap: 3.125rem;
+}
+
+.navLi {
+  list-style: none;
+  font-size: 1rem;
+  font-family: 'Open Sans', sans-serif;
+}
+
+.nextLink {
+  text-decoration: none;
```

Comment:
do you mind adding this to the global styles for anchor tags? This will help us avoid repetition later on.